dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -398,8 +397,6 @@ class ScriptsControllerTest < ActionController::TestCase
     script = Script.find_by_name('test-script-create')
     assert_equal 'test-script-create', script.name
     assert script.has_lesson_plan?
-
-    File.unstub(:write)
```

Comment:
Removing these because they led me toward a bad pattern in my first iteration of this PR.  From the documentation for [`Mocha's `unstub` method](https://github.com/freerange/mocha/blob/d336f49f13a13fdb039fc18497004d2f13c0e72a/lib/mocha/object_methods.rb#L135):

> This is normally done automatically at the end of each test, but in some circumstances you may want to do it *before* the end of the test.

That means these calls at the end of the test are redundant.